django-stubs/db/models/base.pyi

```diff
@@ -1,5 +1,5 @@
 from collections.abc import Collection, Iterable, Sequence
-from typing import Any, TypeVar
+from typing import Any, Final, TypeVar
```

Comment:
Looks like you need to import from `typing_extensions` for Python 3.7.

Reply:
Good catch - thanks!